portal-api/src/v1/facility-dates/timestamp.ts

```diff
@@ -0,0 +1,28 @@
+import { formatISO, set, startOfDay } from "date-fns";
+
+// TODO: DTFS2-6994 remove this function. It seems to be unnecessary with date-fns
+export const formattedTimestamp = (timestamp: string) => {
+  const date = new Date(timestamp);
+  return formatISO(date);
```

Comment:
This is actually a breaking change & needs some more thought.
The 3 all use slightly different expressions for ISO 8601 date:
```js
moment().format();
// 2024-01-29T10:13:49+00:00 
formatISO(new Date());
// 2024-01-29T10:13:49Z
new Date().toISOString();
// 2024-01-29T10:13:49.804Z
```